samples/stopwatch-coroutines/src/main/java/com/instacart/formula/stopwatch/StopWatchViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.instacart.formula.stopwatch
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.instacart.formula.coroutines.toFlow
+import kotlinx.coroutines.flow.*
+
+class StopWatchViewModel(private val formula: StopwatchFormula = StopwatchFormula()) : ViewModel() {
+
+    val rendererStream by lazy {
```

Comment:
Generally, we want to keep the formula running until the screen is destroyed. It seems with `SharingStarted.WhileSubscribed`, configuration changes would reset the state which is not what we usually want.

Reply:
Does this mean that a formula that reacts to a stream of events like the `NetworkStream` sample will keep the producer stream alive even if the Fragment is not visible?

Reply:
Yes, it would keep it alive. You could add logic within `Formula` to optimize expensive operations to detach
```kotlin
if (state.isFragmentInFocus) {
    NetworkStream().onEvent {}
}
```